.github/workflows/doxygen-check.yml

```diff
@@ -1,24 +0,0 @@
-name: Doxygen Check
-
-on:
-  push:
-    branches:
-      - main
-  pull_request:
-    branches:
-      - main
-  workflow_dispatch:
-jobs:
-  doxygen-check:
-    name: Doxygen Check
-    runs-on: self-hosted
-    container:
-      image: "localhost:5000/kamping-ci"
-    steps:
-      - uses: actions/checkout@v2
-      - name: version
-        run: doxygen --version
-      - name: cmake
-        run: cmake -B build
-      - name: run-doxygen
-        run: cmake --build build/ --target docs
```

Comment:
Didn't we have docs for kasser?

Reply:
Future PR, depends on Florians PR

Reply:
Right, so just to have the tag: @kurpicz please add this back in  #2 ;)